Google.PowerShell/Logging/GcLogCmdlets.cs

```diff
@@ -145,6 +145,27 @@ protected MonitoredResourceDescriptor GetResourceDescriptor(string descriptorTyp
         protected string[] AllResourceTypes => s_monitoredResourceDescriptors.Value.Select(descriptor => descriptor.Type).ToArray();
 
         /// <summary>
+        /// Generate -ResourceType dynamic parameter.
```

Comment:
Please elaborate on what -ResourceType means. Such as providing a single example of a cmdlet which would call this method, and why. (Since "ResourceType" is super generic.)

The code describes it on line 155, but if you copied it into the XML Doc comment, it would be even better.